static/js/services/enketo.js

```diff
@@ -18,6 +18,8 @@ angular.module('inboxServices').service('Enketo',
     'ngInject';
 
     var objUrls = [];
+    var xmlCache = {};
+    var FORM_ATTACHMENT_NAME = 'xml';
```

Comment:
Is this constant used elsewhere?  It doesn't seem to be, but presumably the attachment is named elsewhere.  I'd be happy just writing `'xml'` directly below where `FORM_ATTACHMENT_NAME` is used - it's probably better than having a constant which isn't used in all the places it should be.